programs/stake/src/lib.rs

```diff
@@ -12,9 +13,11 @@ const STAKE_PROGRAM_ID: [u8; 32] = [
     120, 114, 43, 104, 164, 157, 192, 0, 0, 0, 0,
 ];
 
-solana_sdk::solana_name_id!(
+solana_sdk::solana_program!(
```

Comment:
This macro can use a better name. Maybe `declare_program!()`

Reply:
Should we also change the `solana_name_id` macro's name as well?